pkgs/applications/misc/OSCAR/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, mkDerivation, fetchFromGitLab, qmake, qtbase, qttools, qtserialport, libGLU }:
+let version = "1.2.0";
+ in
+mkDerivation {
+  name = "OSCAR-${version}";
+  src = fetchFromGitLab {
+    owner = "pholy";
+    repo = "OSCAR-code";
+    rev = "v${version}";
+    sha256 = "10r37d8c2avr167n2s9lhld1c9hmckm444fq163z1jsy9jpid6mg";
+  };
+  buildInputs = [ qtbase qttools qtserialport libGLU ];
+  nativeBuildInputs = [ qmake ];
+  postPatch = ''
+    substituteInPlace oscar/oscar.pro --replace "/bin/bash" "${stdenv.shell}"
```

Comment:
You can probably use patchShebangs here.

Reply:
In this particular case, it isn't a shebang: https://gitlab.com/pholy/OSCAR-code/-/blob/master/oscar/oscar.pro#L76-78